### DISCUSSION TITLE
SlingCatcher SC100 compatibility?

### DISCUSSION THREAD

**Discussion by fridayshoes** [GerryDazoo/Slinger#113] [Q&A]
Would anyone know if it's possible to point the SlingCatcher at @GerryDazoo 's sling server?

My SlingCatcher is running firmware 1.0.0.64 (not sure if this is the latest version)

**Comment by sp222222** [comment]
I had one of those and returned it early on. Didn't like the overall experience using it. I wished I kept it and sold it for the outlandish prices they were going for back then..... I'm not sure it's possible or worth the effort. even a simple mirroring from phone to roku does the trick now.
